packages/esm-patient-forms-app/src/forms/forms-list.test.tsx

```diff
@@ -36,6 +45,10 @@ it('renders a list of forms fetched from the server', async () => {
     showConfigurableForms: false,
   }));
 
+  mockedUserHasAccess.mockImplementation((privilege) => {
```

Comment:
So, we should have some sort of `expect` in the body of this test that asserts that:

1. At least one of the matching forms is rendered
2. At least one of the non-matching forms is not rendered.

Reply:
I have added `expect` blocks, but, neglecting the recent changes I have made, all the mocked forms seem not to be rendered in the table. Only three forms were rendered.
![Screenshot from 2023-03-21 18-03-52](https://user-images.githubusercontent.com/104269786/226666246-ad04c8e0-db10-459c-b888-d775add25a7e.png)

After checking, the `published: false` seemed to be causing the issue. On changing to true for those forms that were not displaying, only one of the forms changed was displayed. I hope this helps in any way.